gen3-integration-tests/services/graph.py

```diff
@@ -235,3 +253,193 @@ def query_node_count(self, node_name: str) -> dict:
         """
         query = f'{{ _{node_name}_count (project_id: "{self.project_id}") }}'
         return self.graphql_query(query)
+
+    def get_path_with_file_node(self, path_to_file=False, file_node=False):
+        """
+        Get all nodes except node with _file category or get node with _file category
+
+        Args:
+            path_to_file: returns all nodes except for node with _file category
+            file_node : returns the node with _file category
```

Comment:
This docstring is confusing
What are the parameter types? sounds like bools?
The function name "get_path_with_file_node" implies that the function returns a path that includes the file node. That is not what the function does